plugins/mcreator-localization/help/ru_RU/villagertrades/price2.md

```diff
@@ -0,0 +1 @@
+Второй (необязательный) элемент обмена.
```

Comment:
Why didn't you use my translation? I don't use translation services but translate the UI tips by heart (myself) so that I keep the initial meaning. The services translate each word separately, not the full phrase, avoid using them.
```suggestion
Второй предмет, который можно использовать при заключении сделки.
```